src/blog/2024/08/dashboard-new-layout-widgets-and-gauges.md

```diff
@@ -0,0 +1,74 @@
+---
+title: New Tab Layout, a Widget and Gauges are Available in FlowFuse Dashboard
+subtitle: Our latest update for FlowFuse Dashboard introduces Tabs, a new Number Input widget, and fresh gauges like Battery and Tank Level, along with much more.
+description: Our latest update for FlowFuse Dashboard introduces Tabs, a new Number Input widget, and fresh gauges like Battery and Tank Level, along with much more.
+date: 2024-08-28
+authors: ["gayan-sandamal"]
+image: /blog/2024/07/images/tile-dashboard-new-charts.png
```

Comment:
I will make this PR ready to review by updating with the latest banner image once @sumitshinde-84 designed

Reply:
Marking this as "Ready for Review" as I received the banner image